Cargo.toml

```diff
@@ -14,3 +14,7 @@ opt-level = 2
 curve25519-dalek-derive = { path = "./curve25519-dalek-derive" }
 curve25519-dalek = { path = "./curve25519-dalek" }
 x25519-dalek = { path = "./x25519-dalek" }
+
+# https://github.com/rust-random/rand/pull/1697
+rand = { git = "https://github.com/rust-random/rand", rev = "99ae06fb348e5c5fee0fe41d2201bd4cbb107635" }
+getrandom = { git = "https://github.com/rust-random/getrandom.git", rev = "f67c70110f706f80d7dd6fa52e38d0920754fe2c" }
```

Comment:
Could go either way on waiting for these to be merged. Right now the build is broken on `main`, so I feel like getting it fixed would probably be for the best, even if we temporarily have some patches to do it.